tests/test_immutableset.py

```diff
@@ -222,6 +222,28 @@ def test_singleton_index(self):
         with self.assertRaises(IndexError):
             s[-25]
 
+    def test_singleton_slice(self):
+        s = ImmutableSet.of([1])
```

Comment:
Using `ImmutableSet.of` is deprecated, use the `immutableset` module-level function instead.